src/analysis/retail/priest/shadow/modules/Abilities.tsx

```diff
@@ -118,6 +93,10 @@ class Abilities extends CoreAbilities {
         gcd: {
           base: 1500,
         },
+        insanityCost:
+          5000 -
+          combatant.getTalentRank(TALENTS.MINDS_EYE_TALENT) * 5 +
+          combatant.getTalentRank(TALENTS.DISTORTED_REALITY_TALENT) * 5,
```

Comment:
is the scale on these correct? that would be 0.1% cost reduction per talent point. looks like you actually want `* 500` to reduce the cost by 5 insanity per.

Reply:
You are correct, I have updated this